contributors/guide/pull-requests.md

```diff
@@ -0,0 +1,283 @@
+# Pull Request Process
+
+This doc explains the process and best practices for submitting a pull request to the [Kubernetes project](https://github.com/kubernetes/kubernetes) and its associated subrepositories. It should serve as a reference for all contributors, and be useful especially to new and infrequent submitters.
+
+- [Before You Submit a Pull Request](#before-you-submit-a-pr)
+  * [Run Local Verifications](#run-local-verifications)
+- [The Pull Request Submit Process](#the-pr-submit-process)
+  * [The Testing and Merge Workflow](#the-testing-and-merge-workflow)
+  * [Marking Unfinished Pull Requests](#marking-unfinished-pull-requests)
+  * [Comment Commands Reference](#comment-commands-reference)
+  * [Automation](#automation)
+  * [How the e2e Tests Work](#how-the-e2e-tests-work)
+- [Why was my pull request closed?](#why-was-my-pr-closed)
+- [Why is my pull request not getting reviewed?](#why-is-my-pr-not-getting-reviewed)
```

Comment:
for these two section title lines suggest Pull Request...with caps to be consistent with the other changes